packages/lit-dev-content/site/tutorials/content/first-component/07.md

```diff
@@ -0,0 +1,74 @@
+In this section we will dynamically change the color of the `word-viewer` when
+the play direction changes.
+
+The [`classMap`](https://lit.dev/docs/templates/directives/#classmap) directive
+assigns a list of classes to an element based on an object. Each key in the
+object is treated as a class name, and if the value is truthy, the class is
+added to the element.
+
+Add the `backwards` class to the `<pre>` element when `this.playDirection` is
+`-1`.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  render() {
+    ...
+    return html`<pre
+      class="${classMap({ backwards: this.playDirection === -1 })}"
+      ...
+    >${word}</pre>`;
+  }
+
+```
+
+```js
+// word-viewer.js
+
+  render() {
+    ...
+    return html`<pre
+      class="${classMap({ backwards: this.playDirection === -1 })}"
+      ...
+    >${word}</pre>`;
+  }
+
+```
+
+{% endswitchable-sample %}
+
+Now when `this.playDirection === -1`, the `<pre>` element will gain the class
+`backwards`. Add a selector and styles for `.backwards` in the static styles.
```

Comment:
New paragraph for the command "Add a selector"

Also suggest omitting the word `static` here